### DISCUSSION TITLE
24차 모임 (2024-11-26) 발표 주제 제안

### DISCUSSION THREAD

**Discussion by haksungjang** [OpenChain-KWG/meeting#8] [General]
안녕하세요.

OpenChain Korea Work Group 24차 모임이 2024년 11월26일 오후2시부터 LG AI연구원(마곡)에서 예정되어 있습니다.

이 때 주제 발표를 원하시는 분은 발표 제목과 간단한 내용을 comment로 남겨주시기 바랍니다.
혹시, (내가 발표하지는 못하지만) 누군가가 공유해주기를 원하시는 주제가 있어도 comment 남겨주세요.

주제 발표 세션과 더불어 Lightening Talk를 시도해보려고 합니다.
Lightening Talk는 특정 주제에 대해 3~5분 가량 가볍게 소개하는 시간입니다. 
Lightening Talk로 다루고자 하는 주제가 있다면 comment 남겨주시기 바랍니다.

감사합니다.

**Comment by haksungjang** [comment]
10/30~31에 예정된 Open Compliance Summit 2024(https://events.linuxfoundation.org/open-compliance-summit/ )에 참석하는 분께서 주요 내용을 요약 발표를 해주시면 좋겠습니다. @JustinWonjaePark

**Reply by JustinWonjaePark** [reply]
😬

**Reply by jungsp** [reply]
원재님 발표 기대기대요~~^^

**Reply by haksungjang** [reply]
박원재, 이서연, 장학성 3인 발표 예정
시간: 40분

**Comment by haksungjang** [comment]
유경재님, 안녕하세요. 

4분기 모임 장소 제공 감사합니다!
이번 모임에서 LG AI연구원의 오픈소스 거버넌스 사례에 대해 세션 발표해 주실 수 있나요? 

초거대 AI 모델을 만들면서 오픈소스 관리 체계를 어떻게 구축하고 있는지, 또 고민하시는 바를 공유하고, 함께 논의하는 시간이 되면 좋을 것 같습니다.

감사합니다. :)

**Reply by jungsp** [reply]
오~ 매우 기다려지네요.

**Reply by KJ-YOO** [reply]
당사 Responsible AI 정책이나 실무 관련하여 소개해보겠습니다. 저랑 같은 팀이기도 하고 업무도 엮여있긴 한데 우선적으로 그 업무를 메인으로 하시는 분을 섭외해보고, 안되면 자료라도 뺏어오도록 하겠습니다.

**Reply by haksungjang** [reply]
제목: LG AI Research의 Responsible AI 정책 
시간: 20분

어떠신가요? @KJ-YOO

**Comment by jungsp** [comment]
저는 이거 한번 발표해보고 싶습니다. EU CRA(Cyber Resilience Act). 아~ 자꾸 손들면 안 되는데 ㅋㅋㅋㅋ

**Reply by haksungjang** [reply]
너무 좋은 주제입니다. 
요건 관련 기사 (이미 보셨겠지만 ㅎㅎ) :  https://www.ddaily.co.kr/page/view/2024101717410413081

**Reply by jungsp** [reply]
네, 다들 CRA를 상세하게 들여다 보시는 분은 없으신 것 같아서 미미하지만 도움될 것 같다는 생각이 들었습니다.ㅎ

**Reply by haksungjang** [reply]
@jungsp 님, 

CRA내 오픈소스 관련 내용에 대해서는 Open Compliance Summit 2024 Review 발표에서 주요 내용을 언급하기로 하고, 시간 관계 상 별도 세션은 갖지 않기로 하였습니다. 

자원해주셔서 감사합니다!!!

**Comment by haksungjang** [comment]
Open Source AI Definition에 대해 누군가 간략히 다뤄주시면 좋을 것 같습니다. https://opensource.org/ai/open-source-ai-definition
- 왜 메타의 라마는 Open Source AI Definition에 따르면 오픈소스가 아니라는 주장이 나오는 걸까?

**Comment by haksungjang** [comment]
독일에서 AI 관련 저작권 판결이 있었습니다. 이에 대해 가볍게 다루면 재밌을 것 같습니다.
- https://jbb.de/to-mine-or-not-to-mine/

**Reply by haksungjang** [reply]
@jungsp 님 comment: 

오케이. 제가 정리하겠습니다.
박정숙 드림.

**Reply by haksungjang** [reply]
넵, 독일 AI 판결 관련해서 작성하고 있는 블로그도 곧 오픈하겠습니다. 감사합니다. @jungsp 센터장님. ^^

**Reply by haksungjang** [reply]
블로그: [To Mine or Not To Mine: 독일 법원이 AI 시대의 저작권 딜레마에 내린 판결](https://openchain-project.github.io/OpenChain-KWG/blog/2024/11/06/to-mine-or-not-to-mine-%EB%8F%85%EC%9D%BC-%EB%B2%95%EC%9B%90%EC%9D%B4-ai-%EC%8B%9C%EB%8C%80%EC%9D%98-%EC%A0%80%EC%9E%91%EA%B6%8C-%EB%94%9C%EB%A0%88%EB%A7%88%EC%97%90-%EB%82%B4%EB%A6%B0-%ED%8C%90%EA%B2%B0/)

**Reply by haksungjang** [reply]
이 내용은 '오픈소스 담당자가 알아야 할 AI의 저작권 이슈' (정윤환) 세션에서 간단히 다루기로 하였습니다.

**Comment by jungsp** [comment]
오케이. 제가 정리하겠습니다.
박정숙 드림.박정숙&nbsp;Jeongsook Park오픈소스센터&nbsp;/ 센터장34129 대전시 유성구 가정로 218Tel 042-860-5468&nbsp;Mobile 010-8849-6513&nbsp;Email&nbsp; ***@***.*** ***@***.***

-----Original Message-----From: Haksung Jang (장학성) ***@***.***&gt;To: OpenChain-KWG/meeting ***@***.***&gt;Cc: jungsp ***@***.***&gt;, Comment ***@***.***&gt;Sent: 2024-11-05T14:44:03+09:00Subject: Re: [OpenChain-KWG/meeting] 24차 모임 (2024-11-26) 발표 주제 제안 (Discussion #8)

독일에서 AI 관련 저작권 판결이 있었습니다. 이에 대해 가볍게 다루면 재밌을 것 같습니다.


https://jbb.de/to-mine-or-not-to-mine/ https://jbb.de/to-mine-or-not-to-mine/

**Comment by haksungjang** [comment]
OpenChain AI Study Group에 대해 소개해주실 분 있나요? 
- OpenChain AI Study Group은 뭐하는 곳이지? 
- 우리 회사에 도움이 될 만한 내용들이 있나?

**Comment by haksungjang** [comment]
마찬가지로, OpenChain SBOM Study Group에 대해 소개해주실 분 있나요? 
- OpenChain SBOM Study Group은 뭐하는 곳이지?
- 우리 회사에 도움이 될 만한 내용들이 있나?

**Comment by daraeahn** [comment]
제가 다른 자리에서 고려대 최윤성 교수님 뵐 기회가 있었는데요, SBOM관련해서 유익한 이야기 해주실 수 있을 것 같아 한번 슬쩍 말씀드려봤는데 좋다고 하셨어요~ 저희 모임에 단순 초청도 좋고 발표해주셔도 좋을거 같은데 어떨까요?

**Reply by haksungjang** [reply]
오, 의견 감사합니다! 운영위원회 모임 시 논의해보시죠~~~ ^^

감사합니다.

**Reply by haksungjang** [reply]
SBOM에 대해서는 'Open Compliance Summit 2024 Review' 세션에서 주요 내용을 다룰 예정이며, 시간 관계 상 별도 세션으로 추가하지는 않기로 하였습니다. 추천해주셔서 감사합니다. 다음 기회에 다시 고려해보면 좋겠습니다. 
감사합니다.

**Comment by haksungjang** [comment]
오픈소스 담당자가 알아야 할 AI의 저작권 이슈 - 정윤환

**Reply by haksungjang** [reply]
발표시간: 30분
